cluster/images/etcd/rollback/etcd237/pkg/fileutil/fileutil.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2015 CoreOS, Inc.
```

Comment:
hmm... this modified wal pkg only lives here now. I did a few modification to keep the dependencies minimal. I am not sure how to solve it. We do not want to have this thing in etcd repo.